### PR TITLE
refactor(vocabulary): rename "export tool" => "translation tool"

### LINKED ISSUE #152: Confusion regarding terms: GUI tool vs. Design tool & Translation tool vs. Export tool
Hello,

_First of all, a huge thank you to all the people who have contributed so far to this wonderful initiative.
Being passionate about UI and standardization, I loved reading the second draft and I want to be part of the contributors by proposing some changes that I hope you will like._

## Explanation

When I read the second draft, I wondered why we use several terms to mean the same thing and suggest you streamline this into one term. 

## Proposals

- rename all "GUI tool" => "Design tool".
- rename all "Export tool" => "Translation tool

What do you think about them?

Thanks in advance for your help!

### PR DESCRIPTION
**Pull request by lauthieb**
In order to be more consistent, I've proposed to align terms of "export tool" & "translation tool".

So, this PR is there to close my issue about this: closes #152

Btw, thanks a lot @c1rrus for your comment on this:

> Good point!
> 
> I think you're right, we've used multiple terms to mean the same thing and ought to clean that up.

Feel free if you want to challenge :)

Have a nice day,

### CODE REVIEW

**Comment by lauthieb** [comment]
Hi @kaelig, hope you're well :)
What will be the next step for this proposal?
Thanks a lot!

File: technical-reports/format/groups.md [hunk 1]
```diff
@@ -127,8 +127,8 @@ For example:
 Suggested ways tools MAY use this property are:
 
 - A style guide generator could render a section for each group and use the description as an introductory paragraph
-- A GUI tool that lets users browse or select tokens could display this info alongside the corresponding group or as a tooltip
```
Comment (c1rrus) [inline technical-reports/format/groups.md:130]: I would leave GUI as it is here. This advice applies to any tool that has a graphical user interface, regardless of whether it is a (UI) design tool or not.

File: technical-reports/format/groups.md [hunk 2]
```diff
@@ -228,15 +228,15 @@ For example:
 
 </aside>
 
-### GUI tools
+### Design tools
```
Comment (c1rrus) [inline technical-reports/format/groups.md:231]: Same comment as above. I'd prefer to keep this as "GUI tools" as this advice can apply to tools that are not design tools but still have a GUI.

**Review by c1rrus — CHANGES_REQUESTED** [review]
Apologies for the long delay before reviewing this (life got in the way).

Thank you so much for taking the time to clear up the "translation tool" / "export tool" inconsistencies! However, I think a few of the other changes should be reverted before this can be approved and merged.

File: technical-reports/format/groups.md (hunk 2 above)
Comment (lauthieb) [inline technical-reports/format/groups.md:231]:
```suggestion
### GUI tools
```

File: technical-reports/format/groups.md [hunk 3]
```diff
@@ -127,8 +127,8 @@ For example:
 Suggested ways tools MAY use this property are:
 
 - A style guide generator could render a section for each group and use the description as an introductory paragraph
-- A GUI tool that lets users browse or select tokens could display this info alongside the corresponding group or as a tooltip
-- Export tools could output this as a source code comment
+- A Design tool that lets users browse or select tokens could display this info alongside the corresponding group or as a tooltip
```
Comment (lauthieb) [inline technical-reports/format/groups.md:130]:
```suggestion
- A GUI tool that lets users browse or select tokens could display this info alongside the corresponding group or as a tooltip
```

**Review by lauthieb — COMMENTED** [review]
Hi @c1rrus, thanks a lot for your review!
Just reverted to "GUI tool" :)

**Review by c1rrus — APPROVED** [review]
Lovely! Thanks so much for making those amends!

**Pull request merged**